linux/ibus-keyman/tests/meson.build

```diff
@@ -0,0 +1,158 @@
+test_files = [
+  'ibusimcontext.c',
+  'ibusimcontext.h',
+  'testfixture.cpp',
+  'testmodule.c',
+  'testmodule.h',
+]
+
+kmnkbp_tests_lib = cc.find_library(
+  'libkmnkbp-tests',
+  dirs: [
+    '@0@/../../../core/build/arch/debug/tests/unit/kmx'.format(meson.current_source_dir()),
+    '@0@/../../../core/build/arch/release/tests/unit/kmx'.format(meson.current_source_dir())
+  ]
+)
+
+deps = [ibus, gtk, x11, json_glib, kmnkbp_lib, kmnkbp_tests_lib]
+
+test_include_dirs = [
+  core_include_dirs,
+  include_directories('../../../core/tests/unit/kmx'),
```

Comment:
```suggestion
  include_directories(core_dir / 'tests/unit/kmx'),
```

Reply:
Unfortunately this doesn't work - meson complains about absolute paths:

```
ERROR: Tried to form an absolute path to a source dir. You should not do that but use relative paths instead.
```